services/ui-src/src/components/logins/LoginCognito.tsx

```diff
@@ -40,30 +40,34 @@ export const LoginCognito = () => {
       <Heading mb="2" size="md" alignSelf="center">
         Login with Cognito
       </Heading>
-      <Heading mb="2" size="sm">
-        Email
-      </Heading>
-      <Input
-        id="email"
-        name="email"
-        type="email"
-        value={fields.email}
-        onChange={handleFieldChange}
-        className="field"
-      />
-      <Heading mb="2" size="sm">
-        Password
-      </Heading>
-      <Input
-        id="password"
-        name="password"
-        type="password"
-        value={fields.password}
-        onChange={handleFieldChange}
-        className="field"
-      />
+      <label>
```

Comment:
Blocking: As far as I understand the a11y documentation, headings should not be nested under label's as it can break some navigation aids. Thus I think we need to either please the label inside the heading and than target the input or need to use something other than an heading for the label styling.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#headings

Reply:
Ah yeah good catch. I'll swap it out.